pkgs/by-name/pr/proton-ge-rtsp-bin/package.nix

```diff
@@ -9,13 +9,19 @@
 proton-ge-bin.overrideAttrs (
   finalAttrs: _: {
     pname = "proton-ge-rtsp-bin";
-    version = "GE-Proton9-20-rtsp16";
+    version = "GE-Proton9-22-rtsp17";
 
     src = fetchzip {
       url = "https://github.com/SpookySkeletons/proton-ge-rtsp/releases/download/${finalAttrs.version}/${finalAttrs.version}.tar.gz";
       hash = "sha256-iq7oiDW5+51wzqYwASOGSV922c/pg1k29MdkIXlT34k=";
     };
 
+    postBuild = ''
+      # prevents steam from resetting compatability settings (in addition to upstream's modifications)
+      sed -i -r 's|GE-Proton-rtsp[0-9]*|GE-Proton-rtsp|' $steamcompattool/compatibilitytool.vdf
```

Comment:
I'd prefer this to use substituteInPlace. You would have to change it from being a regex to a regular string substitution using the metadata we already have in the derivation

Reply:
The upstream package in nixpkgs is already modifying it in the same way. This seems like a simpler way so we don't have to bookkeep the correct rtsp version

Reply:
But anyway, changed

Reply:
Nvm, regex doesn't work with substituteInPlace, or I don't know how to correctly use it if there is a way

Reply:
Does this `sed` even work?

The file shows the following display name in my local copy: `GE-Proton9-22-rtsp17`. Quite conveniently, this is the exact value of `version`. We should be able to use `substituteInPlace` here